pandas/io/s3.py

```diff
@@ -16,8 +16,8 @@ def _strip_schema(url):
     return result.netloc + result.path
 
 
-def get_fs():
-    return s3fs.S3FileSystem(anon=False)
+def get_fs(anon: bool = False):
```

Comment:
These changes will be removed by: https://github.com/pandas-dev/pandas/pull/34266 and reverts error originally introduced in: https://github.com/pandas-dev/pandas/pull/33632/files#diff-a37b395bed03f0404dec864a4529c97dL37